activemodel/test/cases/errors_test.rb

```diff
@@ -228,6 +228,15 @@ def test_no_key
     assert_not person.errors.added?(:name)
   end
 
+  test "added? returns false when checking for an error with an incorrect or missing option" do
+    person = Person.new
+    person.errors.add :name, :too_long, count: 25
+
```

Comment:
Could we please also check that it returns `true` as expected?

```suggestion
assert person.errors.added? :name, :too_long, count: 25
```

Reply:
👍